plugin/trino-cassandra/pom.xml

```diff
@@ -15,6 +15,7 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <dep.native-protocol.version>1.5.1</dep.native-protocol.version>
+        <air.test.parallel>instances</air.test.parallel>
```

Comment:
Was this property required to fix CI failure?

Reply:
Yes this property fixed the CI failure.